libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -325,6 +325,12 @@ void YamlConfiguration::parseRemoteProcessGroupYaml(YAML::Node *rpgNode, core::P
                 }
               }
             }
+          } else if (transport_protocol == "RAW") {
+            group->setTransportProtocol(transport_protocol);
+          } else {
+            std::stringstream stream;
+            stream << "Invalid transport protocol " << transport_protocol;
+            throw minifi::Exception(ExceptionType::SITE2SITE_EXCEPTION, stream.str().c_str());
```

Comment:
This is fine, however makes me wonder why don't we have string param ctr. for minifi::Exception as we convert to the const char* to string anyway in the current ctr.

Reply:
I think it's vestigial (existed since the early days of the project )-- no reason, just no desire to make changes that require unnecessary classes to change in this PR. If someone said, "it has to be done to approve," I'd go ahead and do it but a follow on PR like the one you made is perfect.